modules/nw-sriov-configuring-device.adoc

```diff
@@ -64,13 +78,23 @@ If you specify both `pfNames` and `rootDevices` at the same time, ensure that th
 <10> Optional: Specify the device hex code of SR-IOV network device. The only allowed values are `158b`, `1015`, `1017`.
 <11> Optional: The parameter accepts an array of one or more physical function (PF) names for the Ethernet device.
 <12> The parameter accepts an array of one or more PCI bus addresses for the physical function of the Ethernet device. Provide the address in the following format: `0000:02:00.1`.
+ifdef::ocp-sriov[]
 <13> Optional: Specify the driver type for the virtual functions. You can specify one of the following values: `netdevice` or `vfio-pci`. The default value is `netdevice`.
 +
 [NOTE]
 ====
 For a Mellanox card to work in Data Plane Development Kit (DPDK) mode on bare metal nodes, use the `netdevice` driver type and set `isRdma` to `true`.
-For a Mellanox card to work in DPDK mode with Container-Native Virtualization (CNV), use the `vfio-pci` driver type and set `isRdma` to `false`.
+For a Mellanox card to work in DPDK mode with OpenShift virtualization, use the `vfio-pci` driver type and set `isRdma` to `false`.
```

Comment:
This is the OCP part of the module, which doesn't have a {VirtProductName} attribute, so it has to be hand-written.

@vikram-redhat Should OpenShift adjacent products be added to common-attributes.adoc?

Reply:
> This is the OCP part of the module, which doesn't have a {VirtProductName} attribute, so it has to be hand-written.
> 
> @vikram-redhat Should OpenShift adjacent products be added to common-attributes.adoc?

I am wary of that. While it might not seem that there are too many attributes at this time if we start adding more, it might make it more difficult to separate them later. 

But, I am missing something. Why is this reference to Virt product in the OCP section?

Reply:
@vikram-redhat 1) ack
2) It was added by an SME: https://github.com/openshift/openshift-docs/commit/3b717f6dd0197d83ff715cdff00b53cbbdc05064
It makes sense to include this in the OCP docs in case a user is configuring for OCP and also wants to or might want to use it for VMs, right?

Reply:
> @vikram-redhat 1) ack
> 2) It was added by an SME: [3b717f6](https://github.com/openshift/openshift-docs/commit/3b717f6dd0197d83ff715cdff00b53cbbdc05064)
> It makes sense to include this in the OCP docs in case a user is configuring for OCP and also wants to or might want to use it for VMs, right?

An instance of OCP doesn't include CNV automatically so this inclusion in the docs shouldn't be there. This only makes sense in a cluster with CNV installed and we are likely to run into support issues.

@jboxman @zshi-redhat any objections to removing this line from the OCP docs? 

@aburdenthehand we should conditionalize it for appearance in CNV docs only.

Reply:
@vikram-redhat I'm fine with removing this note for CNV or openshift virtualization. 
IIUC, for any SR-IOV deployment on CNV product, user/customer shall strictly follow CNV doc.

One question about CNV doc.
CNV as a separate product, is also included in openshift-docs?

If yes, @jboxman do you think we shall add a note for the whole hardware-settings part to indicate that this is for SR-IOV on baremetal instead of CNV?

Reply:
> One question about CNV doc.
> CNV as a separate product, is also included in openshift-docs?

CNV is not a separate product per se. It is a feature of OpenShift, similar to serverless AFAIK. Just a nit :)

Reply:
@phoracek gotcha, I got the wrong impression that CNV is a product as I heard the term `CNV product` :-)
thought it has own release cadence, not aligned with openshift release. 
Thanks for clarification!

Reply:
Heh :D I don't blame you. It is standalone to a high degree, but the narrative we want to use publicly is that virtualization *is* OpenShift.

Reply:
Thanks for the clarification folks. I've separated the content for <deviceType> and <isRdma> with conditionals so that OCP content focusses on OCP and CNV on CNV.
@jboxman Do you want me to add the suggestion from @zshi-redhat in the /networking/hardware_networks/about-sriov.adoc in this PR?